res_interpreter_language/views/res_interpreter_language_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_interpreter_language_form" model="ir.ui.view">
+            <field name="name">res.interpreter_language.form</field>
+            <field name="model">res.interpreter.language</field>
+            <field name="arch" type="xml">
+                <form string="Interpreter Language">
+                    <sheet>
+                        <!--<h1>
+                            <field name="name" default_focus="1" placeholder="Name" />
+                        </h1> -->
```

Comment:
Please state in a code comment above why you want to keep this code

Reply:
I don't want to keep it, I'll remove.